### PR TITLE
[Rec-IM] Start Legal Requirements on Esignature

### PR DESCRIPTION
**Pull request by bennettforkner**
This PR will add scaffolding for the requirements of a legally-binding e-signature on a waiver according to [this documentation](https://blog.waiverforever.com/electronic-signatures-legally-binding/).

### CODE REVIEW

File: src/views/RecIM/components/Forms/WaiverForm/index.js [hunk 1]
```diff
@@ -1,21 +1,30 @@
...
-      label: 'By Clicking this, I have certified that I have read the Gordon Waiver',
+      label: 'By checking this box, I agree to sign this waiver electronically.',
+      name: 'esignCheckbox',
+      type: 'checkbox',
+      error: errorStatus.esignCheckbox,
+      helperText: '*Required',
+    },
+    {
+      label:
+        'By checking this box, I have certified that I have read the Gordon Waiver. I also agree to sign this waiver electronically.',
```
Comment (bennettforkner) [inline src/views/RecIM/components/Forms/WaiverForm/index.js:27]:
```suggestion
        'By checking this box, I have certified that I have read the Gordon Waiver.',
```

File: src/views/RecIM/views/Home/index.js [hunk 2]
```diff
@@ -184,7 +182,7 @@ const Home = () => {
               handleOpenWaiverForm(status);
             }}
             openWaiverForm={openWaiver}
-            setOpenWaiverForm={(bool) => setOpenWaiver(bool)}
+            setOpenWaiverForm={(waiverOpen) => setOpenWaiver(waiverOpen)}
```
Comment (bennettforkner) [inline src/views/RecIM/views/Home/index.js:185]: naming parameters after a type is not great practice

Comment (amos-cha) [reply src/views/RecIM/views/Home/index.js:185]: I think it happened when I was mass renaming things, but I'll make sure I apply that to future instances!

**Comment by bennettforkner** [comment]
This is no longer relevant. We are going to try and build this elsewhere.

**Comment by amos-cha** [comment]
sounds good!

**Pull request closed without merging**